src/scss/core/_vars.scss

```diff
@@ -7,3 +7,39 @@ $grid-breakpoints: (
   lg: 992px,
   xl: 1200px
 );
+
+$grid-gutter-width: 15px;
+$grid-gutter-width-lg: 30px;
+
+// colors
+$primary: #00635B;
+$secondary: #FF5A8D;
```

Comment:
primary color is the pink, light variants must be always referred to the color var, so light-primary (better primary-light) not light-pink